Source/Core/Core/IOS/Network/Socket.cpp

```diff
@@ -712,24 +709,99 @@ void WiiSockMan::Update()
       socket_iter = WiiSockets.erase(socket_iter);
     }
   }
-  s32 ret = select(nfds, &read_fds, &write_fds, &except_fds, &t);
 
-  if (ret >= 0)
+  // ret is unused as every socket gets updated anyway
+  [[maybe_unused]]
+	  s32 ret = poll(ufds.data(), nfds, 0);
```

Comment:
The return value should be checked and at least an `ERROR_LOG` should be printed when it fails. `WiiSockMan::GetNetErrorCode` might be used here.